app/views/PushTroubleshootView/index.tsx

```diff
@@ -0,0 +1,145 @@
+import { requestTroubleshootingNotification } from '../../actions/troubleshootingNotification';
+import { useAppSelector, usePermissions } from '../../lib/hooks';
+import { Services } from '../../lib/services';
+
+interface IPushTroubleshootViewProps {
+	navigation: StackNavigationProp<SettingsStackParamList, 'PushTroubleshootView'>;
+}
+
+const PushTroubleshootView = ({ navigation }: IPushTroubleshootViewProps): JSX.Element => {
+	const { colors } = useTheme();
+	const dispatch = useDispatch();
+	const [testPushNotificationsPermission] = usePermissions(['test-push-notifications']);
+
+	const { deviceNotificationEnabled, defaultPushGateway, pushGatewayEnabled, foreground, serverVersion } = useAppSelector(
+		state => ({
+			deviceNotificationEnabled: state.troubleshootingNotification.deviceNotificationEnabled,
+			pushGatewayEnabled: state.troubleshootingNotification.pushGatewayEnabled,
+			defaultPushGateway: state.troubleshootingNotification.defaultPushGateway,
+			foreground: state.app.foreground,
+			serverVersion: state.server.version
+		})
+	);
+
+	useEffect(() => {
+		if (foreground) {
+			dispatch(requestTroubleshootingNotification());
+		}
+	}, [dispatch, foreground]);
+
+	useEffect(() => {
+		navigation.setOptions({
+			title: I18n.t('Push_Troubleshooting')
+		});
+	}, [navigation]);
+
+	const openNotificationDocumentation = async () => {
+		await Linking.openURL('https://go.rocket.chat/i/push-notifications');
+	};
+
+	const alertDeviceNotificationSettings = () => {
+		showErrorAlert(
+			I18n.t('Device_notifications_alert_description'),
+			I18n.t('Device_notifications_alert_title'),
+			goToNotificationSettings
+		);
+	};
+
+	const goToNotificationSettings = () => {
+		if (isIOS) {
+			Linking.openURL('app-settings:');
+		} else {
+			notifee.openNotificationSettings();
+		}
+	};
+
+	const handleTestPushNotification = async () => {
+		let message = '';
+		try {
+			const result = await Services.pushTest();
+			if (result.success) {
+				message = I18n.t('Your_push_was_sent_to_s_devices', { s: result.tokensCount });
+			}
+		} catch (error: any) {
+			message = I18n.isTranslated(error?.data?.errorType) ? I18n.t(error?.data?.errorType) : error?.data?.error;
+		} finally {
+			Alert.alert(I18n.t('Test_push_notification'), message);
+		}
+	};
+
+	let pushGatewayInfoDescription = 'Push_gateway_not_connected_description';
+	let pushGatewayStatusColor = colors.userPresenceBusy;
+	if (pushGatewayEnabled) {
+		pushGatewayStatusColor = colors.userPresenceOnline;
+		pushGatewayInfoDescription = 'Push_gateway_connected_description';
+	}
+	if (pushGatewayEnabled && !defaultPushGateway) {
+		pushGatewayStatusColor = colors.badgeBackgroundLevel3;
+		pushGatewayInfoDescription = 'Custom_push_gateway_connected_description';
+	}
+
+	return (
+		<SafeAreaView testID='push-troubleshoot-view'>
+			<StatusBar />
+			<List.Container testID='push-troubleshoot-view-list'>
+				<CustomListSection
```

Comment:
`List` is one of our only well defined design system components 😬 
Usually when there's need to overwrite behavior design systems, we should first understand if there's really a good reason for that and think of improving the component.
We had issue with this on web and fuselage.